### PR TITLE
Initial manual tests for basic card

### PR DESCRIPTION
**Pull request by marcoscaceres**

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PASSED

Started: 2017-08-11 06:24:15
Finished: 2017-08-11 06:29:33



View more information about this build on:

- [WPT Results Dashboard](http://pulls.web-platform-tests.org/build/17252)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/263375285)

**Comment by w3c-bots** [comment]
# Build BROKEN

Started: 2017-08-11 07:46:36
Finished: 2017-08-11 07:56:40





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/17256)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/263392706)

File: payment-method-basic-card/payment-request-canmakepayment-method.https.html [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  supportedMethods: "this-is-not-supported",
+});
+
+promise_test(async t => {
+  const request = new PaymentRequest([notSupportedMethod], defaultDetails);
+  assert_false(
+    await request.canMakePayment(),
+    `canMakePaymentPromise should be false`
+  );
+}, `Must return false when the PMI is not supported at by the user agent.`);
```
Comment (foolip) [inline payment-method-basic-card/payment-request-canmakepayment-method.https.html]: This and the other tests fail as such in Chrome:
promise_test: Unhandled rejection with value: object "TypeError: Failed to construct 'PaymentRequest': The provided value cannot be converted to a sequence."

Chrome bug, or test bug?

**Comment by foolip** [comment]
I came across this because of status:needs-owners label. Can you add OWNERS?

**Comment by marcoscaceres** [comment]
@foolip owners are now there. Will check other issue.

File: payment-method-basic-card/payment-request-canmakepayment-method.https.html (hunk 1 above)
Comment (marcoscaceres) [reply payment-method-basic-card/payment-request-canmakepayment-method.https.html]: Chrome bug in this case.

**Comment by marcoscaceres** [comment]
@rsolomakhin ... should probably add you to OWNERS too. If you have time, could you please take a look at these tests? It's been a while since I wrote/ran them 🙃

File: payment-method-basic-card/payment-request-canmakepayment-method.https.html (hunk 1 above)
Comment (foolip) [reply payment-method-basic-card/payment-request-canmakepayment-method.https.html]: OK, thanks for poking at it with a test!

**Review by foolip — APPROVED** [review]

**Review by rsolomakhin — APPROVED** [review]
These tests look good. Thank you!

P.S.: It's OK to add me to owners.

**Comment by foolip** [comment]
I added @rsolomakhin in https://github.com/w3c/web-platform-tests/pull/7640 and merged both. @rsolomakhin, can you file a  Chromium bug for the failing test mentioned above?

**Comment by rsolomakhin** [comment]
@foolip https://crbug.com/754779

**Pull request merged**